### DISCUSSION TITLE
Would you consider moving this into Flutter Community?

### DISCUSSION THREAD

**Discussion by kvenn** [isar-community/isar#25] [General]
To prevent what happened with the original Isar, it might be better to submit this into the Flutter community where it can live forever. Since this project is still new, this might be the most viable time to pull it off.

I think it's GREAT that this fork happened, I just want it to have the best chance to succeed.

https://github.com/fluttercommunity/community

Thoughts?

**Comment by mrclauss** [comment]
Well, the problem is, if you look at the transfer howto, that you need the original author to add flutter community as an uploader, which we can't do (for the original package).
Additionally the build system of Isar is a bit more complex because of the native rust libraries and the inspector. Shifting to fluttercommunity might make some of those hard to adapt/change/realise...

**Reply by kvenn** [reply]
Okay interesting. I was actually recommending transferring _this_ package into fluttercommunity. As for the fact that there are multiple repos in this organization, I'd imagine they'd go over to fluttercommnity as well. Or they could remain as dependencies in another organization if that's what you prefer.

And all existing contributors could continue contributing as they were, just in a different org.

But I understand it's effort - just trying to prevent isar-community from suffering the similar fate of `isar`. Especially since it could be such a core package in the Flutter ecosystem. Maybe even getting the original maintainer back into it (which seems even more reasonable if it's in fluttercommunity).

Anyways, it's totally your call. Just figured I'd offer the suggestion.
